automation_testing/tests/dataview/data_view.spec.ts

```diff
@@ -1,28 +1,32 @@
 import { test, expect } from "@playwright/test";
+import { ColumnsComponent } from "../../pages/ColumnsComponent";
 import { DataviewPage } from "../../pages/DataViewPage";
 import { dataview } from "../../utils/data/dataview_data";
 
 test.describe("Data View", () => {
 	let dataviewPage: DataviewPage;
+	let columns: ColumnsComponent;
 
 	test.beforeEach(async ({ page }) => {
 		dataviewPage = new DataviewPage(page);
+		columns = dataviewPage.columnsComponent;
+
 		await dataviewPage.visit();
 	});
 
-	test("Create New", async () => {
+	test("DataView - Create New", async () => {
```

Comment:
I feel this might be repeated as the describe of the test already is "Data View", maybe we could remove it and keep it as it was... or was there a specific reason for you to add it?